tools/send-to-caas/send-utils.js

```diff
@@ -589,21 +589,24 @@ const getCardMetadata = async (options) => {
 };
 
 const postDataToCaaS = async ({ accessToken, caasEnv, caasProps, draftOnly }) => {
+  const uie = window.adobeid?.environment?.toLowerCase
+    && window.adobeid.environment.toLowerCase() !== 'prod' ? { 'user-ims-env': 'stage' } : {};
+
   const options = {
     method: 'POST',
     body: JSON.stringify(caasProps),
     headers: {
       Authorization: `Bearer ${accessToken}`,
       draft: !!draftOnly,
       'caas-env': caasEnv,
+      ...uie,
```

Comment:
The whole part can be written as:
```js
'user-ims-env': window.adobeid?.environment?.toLowerCase() === 'prod' ? '' : 'stage',
```

Reply:
I would even pass `'prod'` instead of `''` for the first use-case but now sure whether this is handled in CaaS.

Reply:
Thank you @narcis-radu , In order keep existing call unchanged, we were adding user-ims-env for stage only. Can recheck and add as suggested.